README.md

```diff
@@ -342,29 +343,48 @@ Plug 'autozimu/LanguageClient-neovim', {
 
 and issuing a `:PlugInstall` command within neovim.
 
-Finally, make sure that `hie` is included as the language server source for haskell:
+#### Vim 8.0
+Clone [LanguageClient-neovim](https://github.com/autozimu/LanguageClient-neovim)
+into `~/.vim/pack/XXX/start/`, where `XXX` is just a name for your "plugin suite".
 
-```
-let g:LanguageClient_serverCommands = {
-    ...
-    \ 'haskell': ['hie-wrapper'],
-    ...
-    \ }
+#### Sample `~/.vimrc`
+
+```vim
+set rtp+=~/.vim/pack/XXX/start/LanguageClient-neovim
+let g:LanguageClient_serverCommands = { 'haskell': ['hie-wrapper'] }
 ```
 
-Since LanguageClient-neovim doesn't start language servers in the project root, it may also be helpful to also specify the project root for hie-wrapper, writing instead:
+You'll probably want to add some mappings for common commands:
 
+```vim
+nnoremap go :call LanguageClient#textDocument_documentSymbol()<CR>
+nnoremap ga :call LanguageClient#textDocument_codeAction()<CR>
+nnoremap gd :call LanguageClient#textDocument_definition()<CR>
+nnoremap gr :call LanguageClient#textDocument_rename()<CR>
+nnoremap gfd :call LanguageClient#textDocument_formatting()<CR>
+nnoremap K :call LanguageClient_textDocument_hover()<CR>
```

Comment:
I used to have something like this, but it steals too many of the functions, specially for things that are not using a language server. I came up with something less disruptive, maybe you can consider recommending these instead?

```viml
nnoremap <F5> :call LanguageClient_contextMenu()<CR>
map <Leader>lk :call LanguageClient#textDocument_hover()<CR>
map <Leader>lg :call LanguageClient#textDocument_definition()<CR>
map <Leader>lr :call LanguageClient#textDocument_rename()<CR>
map <Leader>lf :call LanguageClient#textDocument_formatting()<CR>
map <Leader>lb :call LanguageClient#textDocument_references()<CR>
map <Leader>la :call LanguageClient#textDocument_codeAction()<CR>
map <Leader>ls :call LanguageClient#textDocument_documentSymbol()<CR>
```